integrationtest/backend/lucene/src/test/java/org/hibernate/search/integrationtest/backend/lucene/LuceneExtensionIT.java

```diff
@@ -270,12 +276,12 @@ public void query_explain_multipleIndexes_invalidIndexName() {
 				.toQuery();
 
 		Assertions.assertThatThrownBy(
-				() -> query.explain( "NotAnIndexName", FIRST_ID )
+				() -> query.explain( "NotAMappedTypeName", FIRST_ID )
 		)
 				.isInstanceOf( SearchException.class )
 				.hasMessageContaining(
-						"index name 'NotAnIndexName' is not among the indexes targeted by this query: ["
-								+ INDEX_NAME + ", " + OTHER_INDEX_NAME + "]"
+						"mapped type name 'NotAMappedTypeName' is not among the mapped types targeted by this query: [",
+						TYPE_NAME, OTHER_INDEX_NAME, "]"
```

Comment:
`OTHER_INDEX_NAME` => `OTHER_TYPE_NAME`?

I'm surprised that the test passed, by the way...

Reply:
it was my fault, I always thought that `hasMessageContaining` takes more message terms as a parameter, but it isn't so.
To check more terms we have to invoke `hasMessageContaining` more times.
Fixing it...